components/dialog/README.md

```diff
@@ -83,6 +83,12 @@ The `d2l-dialog` element is a generic dialog that provides a slot for arbitrary
 - `d2l-dialog-close`: dispatched with the action value when the dialog is closed for any reason
 <!-- docs: end hidden content -->
 
+### Accessibility Properties
+
+| Attribute | Description |
+|--|--|
+| `describe-content` | When set to 'true' reads the dialog content on open. |
```

Comment:
I might just rephrase this so consumers don't attempt to assign "true".  Maybe `When set, screen readers will announce the contents when opened. Useful when the the dialog contains no other focusable elements.`